nltk/tag/stanford.py

```diff
@@ -204,6 +205,75 @@ def parse_output(self, text, sentences):
 
         raise NotImplementedError
 
+class CoreNLPTagger(CoreNLPParser, TaggerI):
+    def __init__(self, tagtype, url='http://localhost:9000', encoding='utf8'):
+        """
+        An abstract interface to POS/NER taggers of CoreNLP that returns the
+        POS/NER tags from the Stanford CoreNLP API at nltk.parse.corenlp.
+        """
+        self.tagtype = tagtype
+        super(CoreNLPTagger, self).__init__(url, encoding)
+
+    def tag_sents(self, sentences):
+        sentences = (' '.join(words) for words in sentences)
+        return list(self.raw_tag_sents(sentences))
+
+
+    def tag(self, sentence):
+        return self.tag_sents([sentence])[0]
+
+    def raw_tag_sents(self, sentences):
+        """
+        This function will interface the `GenericCoreNLPParser.api_call` to
+        retreive the JSON output and return the annotations required.
+        """
+        default_properties = {'ssplit.isOneSentence': 'true',
+                              'annotators': 'tokenize,ssplit,' }
+        # Supports only 'pos' or 'ner' tags.
+        assert self.tagtype in ['pos', 'ner']
+        default_properties['annotators'] += self.tagtype
+        for sentence in sentences:
+            tagged_data = self.api_call(sentence, properties=default_properties)
+            assert len(tagged_data['sentences']) == 1
+            # Taggers only need to return 1-best sentence.
+            yield [(token['word'], token[self.tagtype]) for token in tagged_data['sentences'][0]['tokens']]
+
+
+class CoreNLPPOSTagger(CoreNLPTagger):
+    """
+    This is a subclass of the CoreNLPTagger that wraps around the
+    nltk.parse.CoreNLPParser for Part-of-Sppech tagging.
+
+        >>> from nltk.tag.stanford import CoreNLPPOSTagger
+        >>> tagged = CoreNLPPOSTagger().tag('What is the airspeed of an unladen swallow ?'.split()) # doctest: +SKIP
```

Comment:
`CoreNLPPOSTagger()` assumes that a server is running on port 9000. I would set the link to it, so that the user sees that corenlp server has to be running.